tests/inference/test_modeling.py

```diff
@@ -136,6 +136,26 @@ def test_save_compiler_intermediary_files(self):
             self.assertTrue(os.path.isdir(save_path))
             self.assertTrue(os.path.exists(neff_path))
 
+    @requires_neuronx
+    def test_decouple_weights_neff_and_replace_weight(self):
+        with tempfile.TemporaryDirectory() as tempdir:
+            # compile
+            save_path = f"{tempdir}/neff"
+            neuron_model = NeuronModelForSequenceClassification.from_pretrained(
+                self.MODEL_ID,
+                export=True,
+                compiler_workdir=save_path,
+                inline_weights_to_neff=False,
+                **self.STATIC_INPUTS_SHAPES,
+            )
+            self.assertFalse(neuron_model.config.neuron.get("inline_weights_to_neff"))
+
+            # replace weights
+            model = AutoModelForSequenceClassification.from_pretrained(self.MODEL_ID)
+            neuron_model.replace_wights(weights=model)
```

Comment:
```suggestion
            neuron_model.replace_weights(weights=model)
```

Reply:
And that's why the CI failed this time, thanks for helping me debug. Guess I deleted a letter randomly while fixing the doc string 😅.